_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,496 @@
+---
+id: efbfc7
+name: Auto-updating text content can be paused or stopped
+rule_type: atomic
+description: |
+  This rule checks that there are mechanisms to pause or stop the auto-updating of text content
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+acknowledgements:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [visible][] [text node][] in the [flat tree][] of an [HTML document][] if:
+
+- the content of the [text node][] changes through an action that was not initiated by the user;
```

Comment:
This seems ambiguous. what does it mean for a user to have initiated something? If I click a button that starts a timer for something to happen a week from now, is that event a week from now something I initiated? In that sense, isn't everything I do on my mac ultimately initiated by me?

Reply:
Rephrased it